src/Nancy.Tests/Unit/AfterPipelineFixture.cs

```diff
@@ -102,5 +103,40 @@ public void Pipeline_containing_another_pipeline_will_invoke_items_in_both_pipel
             Assert.True(item3Called);
             Assert.True(item4Called);
         }
+
+        [Fact]
+        public void Pipeline_containing_method_returning_null_from_method_throws_InvalidOperationException()
+        {
+            pipeline.AddItemToEndOfPipeline(ReturnNull);
+
+
+            var exception = Assert.Throws<InvalidOperationException>(() =>
+            {
+                pipeline.Invoke(CreateContext(), new CancellationToken());
+            });
+
+            Console.WriteLine(exception);
+
+            Assert.Contains("ReturnNull", exception.Message);
+        }
+
+        [Fact]
+        public void Pipeline_containing_method_returning_null_from_lambda_throws_InvalidOperationException()
+        {
+            pipeline.AddItemToEndOfPipeline((context, ct) => null);
+
+            var exception = Assert.Throws<InvalidOperationException>(() =>
+            {
+                pipeline.Invoke(CreateContext(), new CancellationToken());
+            });
+
+            Console.WriteLine(exception);

```

Comment:
The point of this PR is to improve the exception being thrown when a pipeline method returns `null`. Writing the exception to the `Console` is the only way I can think of to actually see the caught exception.